README.md

```diff
@@ -1,6 +1,61 @@
 # zarr-proxy
 
-## Client Examples
+_✨ This code is highly experimental! Let the buyer beware ⚠️ ;) ✨_
+
+| CI          | [![GitHub Workflow Status][github-ci-badge]][github-ci-link] [![Code Coverage Status][codecov-badge]][codecov-link] [![pre-commit.ci status][pre-commit.ci-badge]][pre-commit.ci-link] |
+| :---------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| **Docs**    |                                                                     [![Documentation Status][rtd-badge]][rtd-link]                                                                     |
+| **Package** |                                                          [![Conda][conda-badge]][conda-link] [![PyPI][pypi-badge]][pypi-link]                                                          |
+| **License** |                                                                         [![License][license-badge]][repo-link]                                                                         |
+
+A proxy for zarr stores that allows for chunking overrides. This is useful for clients that want to request data in a specific chunking scheme, but the data is stored in a different chunking scheme (e.g. a dataset stored in a chunking scheme that is optimized for fast reading, but the client wants to request data in a chunking scheme that is optimized for fast rendering). One advantage of using a proxy is that we don't need to persistently store the data in multiple chunking schemes. Instead, we can simply request the data in the desired chunking scheme on the fly.
```

Comment:
nit: in the JS client section, I refer to "Zarr" -- can we choose one consistent capitalization scheme?